examples/complex-storage-structures/README.md

```diff
@@ -0,0 +1,340 @@
+# Storage refactoring
```

Comment:
The `examples` folder was renamed to `integration-tests` by now, can you do it in this PR as well?